docs/sources/configuration/_index.md

```diff
@@ -1917,6 +1919,11 @@ The `compactor` block configures the compactor component, which compacts index s
 # CLI flag: -boltdb.shipper.compactor.retention-table-timeout
 [retention_table_timeout: <duration> | default = 0s]
 
+# Store used for managing delete requests. If not set,
+# -boltdb.shipper.compactor.shared-store is used as fallback.
```

Comment:
"as fallback"  Would "default" be a better word choice, since fallback implies something that you use in an emergency?